### ISSUE TITLE
RDotNet.Graphics.WPF

### ISSUE THREAD

**Issue by AndreyAkinshin** [jmp75/rdotnet#11]
The RDotNet.Graphics library looks great, but it is only for WinForms. Do you have plans to implement same feature for WPF?

**Comment by skyguy94** [comment]
RDotNet.Graphics is for creating graphics devices used by R. There's nothing WinForms specific, or anything that precludes from using it in WPF. I've implemented a svg device and a png device for it that  could be used by the appropriate converters/and or image elements.

**Comment by AndreyAkinshin** [comment]
Ok, It get it. In this case, what do you think abount a WPF sample of RDotNet.Graphics using?

**Comment by skyguy94** [comment]
I I'd have to look at the onboarding project and find some time. You just want to embed the plots? As an alternative to RDotNet, have you thought about using R to save to a file and then loading that file?

**Comment by skyguy94** [comment]
....I wish we had a gitter room for discussions like this.

**Comment by AndreyAkinshin** [comment]
Yes, I want to embed plots. In the perfect case, I want to do it without I/O operations because I have big amout of images and I want to do it fast.

**Comment by skyguy94** [comment]
Ok, I'd argue that serializing to disk probably isn't slow. RStudio uses that technique for its plots and serving them from its backend via http requests. If I can get some time on Fri, I'll see if I can put together a demo of the svg device in WPF.

**Comment by AndreyAkinshin** [comment]
Ok, thanks.
